lib/wit/gitrepo.py

```diff
@@ -89,6 +95,17 @@ def clone(self, source, name):
             else:
                 raise
 
+    # Only newer git versions can use '--reference-if-able', so we emulate the 'if-able'.
+    def git_reference_capable(self):
+        env_ref_path = os.getenv("WIT_WORKSPACE_REFERENCE")
+        paths = [Path(env_ref_path) / self.name,
+                 Path(env_ref_path) / (self.name+'.git')]
+        for path in paths:
+            log.warn("{} {}".format(path, os.path.isdir(path)))
+            if os.path.isdir(path):
```

Comment:
Since these are already `Path` objects, you should probably be using the Python 3.4-style [`path.is_dir()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_dir). See https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module for a table that maps from the older `os.path` functions to the newer `Path` methods.